src/careers/current-openings/_single.html

```diff
@@ -23,23 +22,33 @@
     <section class="block
                     block__flush-top
                     block__sub">
-        {# TODO: Add real page details. #}
-        <h1>Test Opening Title</h1>
-        <dl class="u-mb0">
+        <h1>{{ career.title }}</h1>
+        {% set grade = career.grades[0] %}
+        {% from "events/_event-post-macros.html"
+           import event_meta_datetime as event_meta_datetime %}
+        <dl>
             <dt>Expiration Date:</dt>
-            <dd>Month XX, XXXX</dd>
+            <dd>
+                <time datetime="{{ career.close_date | date('%Y-%m-%-dT%H:%M:%S%z') }}">
+                    <span class="event-meta_date">
```

Comment:
Class `event-meta_date` class doesn't seem appropriate for this page, will need to be generalized later. If you need this class for styles I think it's okay for now. If you don't need it, please remove.